VideoLocker/res/layout/row_course_list.xml

```diff
@@ -214,22 +175,42 @@
                 android:layout_height="wrap_content"
                 android:layout_gravity="center_vertical"
                 android:ellipsize="end"
+                android:singleLine="true"
                 android:textColor="@color/org_code_txt"
-                android:textSize="12sp"
-                android:singleLine="true"/>
+                android:textSize="12sp" />
         </LinearLayout>
     </RelativeLayout>
 
-    <!-- put a frame over this cell, so that corners look rounded -->
+    <org.edx.mobile.view.custom.ETextView
+        android:id="@+id/course_certified_banner"
+        style="@style/bold_white_text"
+        android:layout_width="180dp"
```

Comment:
Hmm, as I mentioned in the previous [comment][] on this, we should ideally be using `minWidth` here, with `layout_width` set as before to `wrap_content` to match the previous implementation and allow the view to expand.

[comment]: https://github.com/edx/edx-app-android/pull/354#discussion_r39250531